cypress/integration/ui/view_dashboard/index.js

```diff
@@ -0,0 +1,93 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const antenatalCareDashboardRoute = '#/nghVC4wtyzi'
+const immunizationDashboardRoute = '#/TAMlzYkstb7'
+
+Given('I open the Antenatal Care dashboard', () => {
```

Comment:
Would it be possible to keep `Antenatal Care` in a variable to easily switch it out if the DB changes or we decide its better to use another dashboard as our test dashboard? Or to pick it out of an array to loop through this multiple times for different dashboards?

Reply:
Will revisit this when I add the new set of tests